optional/nothing.py

```diff
@@ -6,10 +6,13 @@ class Nothing(AbstractOptional):
     def is_present(self):
         return False
 
-    def get(self):
-        raise OptionalAccessOfEmptyException(
-            "You cannot call get on an empty optional"
-        )
+    def get(self, default_value=None):
+        if default_value is not None:
```

Comment:
This is the reason why we shouldn't do this. It's not great that we can't meaningfully do `some_optional.get(None)` if we desire.